android/app/src/main/java/com/example/android/data/ItemNetwork.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.android.data
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object ItemNetwork {
+    val itemController by lazy {
+        Retrofit
+            .Builder()
+            .baseUrl("http://10.0.2.2:8080/")
```

Comment:
**suggestion**: You probably want to make this configurable.

Maybe it could be part of the build config like the username and password?

Reply:
Sounds good